src/instructlab/model/dk_bench_utils.py

```diff
@@ -0,0 +1,320 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=ungrouped-imports
+# Standard
+from datetime import datetime
+from typing import List
+import enum
+import logging
+import os
+import pathlib
+
+# Third Party
+from instructlab.eval.mt_bench_common import (
+    get_openai_client as get_local_openai_client,
+)
+from instructlab.eval.ragas import ModelConfig, RagasEvaluator
+from ragas.evaluation import EvaluationResult  # type: ignore
+import click
+import pandas as pd
+
+# Local
+from ..configuration import DEFAULTS
+from .evaluate import get_model_name as get_local_model_name
+from .evaluate import launch_server, validate_model
+
+logger = logging.getLogger(__name__)
+
+
+class IOFileType(enum.Enum):
+    CSV = "csv"
+    JSONL = "jsonl"
+    XLSX = "xlsx"
+
+
+def validate_output_file_formats(file_formats: List[str]):
```

Comment:
can you add docstrings for all of these functions?

Reply:
Docstrings will be added along with unit tests when there isn't as much change in this PR.